src/components/WeekTabs/WeekTabs.jsx

```diff
@@ -0,0 +1,79 @@
+import { React, useEffect, useState } from "react";
```

Comment:
React should be imported by default. If it works now, probably you dont need to import it at all